### ISSUE TITLE
Support of SlaveOfNoOne Command

### ISSUE THREAD

**Issue by andrey-zhminka** [andrew-bn/RedisBoost#7]
Hello, I am trying to call command SlAVEOF NO ONE as mentioned in doc:  http://redis.io/commands/SLAVEOF, bu I can't figure out how to call it via Redis.Boost. 
Does it supported now or you can implement it?

**Comment by andrey-zhminka** [comment]
Possible solution is like this method:
```
public Task<string> SlaveOfNoOneAsync()
{
	return StatusCommand(RedisConstants.SlaveOf, "NO".ToBytes(), "ONE".ToBytes());
}
```

**Comment by andrew-bn** [comment]
Hi. 

Yes, you are correct. 
That command was not implemented. And your solution is the right one.
I've just pushed the fix.
In case of not implemented commands you can also use ExecuteAsync command.

**Comment by andrew-bn** [comment]
Hi,
New version was released: RedisBoost 1.6.4
